assets/readme.md

```diff
@@ -1,19 +1,121 @@
+![Getting Started Step 1](./docs/gettingstarted01.png)
+Your Clearbit Connector is now ready to be configured depending on your needs, see Features to learn how to configure the various solutions.
 
-## Similar Lead discovery
+## Features
 
-The lead discovery feature uses [Clearbit Discovery](https://clearbit.com/discovery) and [Clearbit Prospector](https://clearbit.com/prospector) to find new leads based your existing users.
+Clearbit [**Enrichment**](#Enrichment) enables your team to transform the email address of an user in Hull into a complete user and company profile with up to 85 data points.
 
-Whenever a new user enters a segment that has been enabled in the "Similar lead discovery" section, this ship will :
+With Clearbit [**Reveal**](#Reveal) you can turn your anonymous website traffic into company profiles to power targeted campaigns and close more named accounts .
 
-1. Look for companies that are similar to the user's company via [Clearbit's Discovery](https://clearbit.com/discovery)
-2. Then for each Company found, we look for Prospect that match a specified role and/or seniority within that Company, via [Clearbit's Prospector](https://clearbit.com/prospector)
-3. For each Prospect found, we create a new User on your organization with the "Cleabit > Prospected at" attribute set to the current date.
+Use the [**Prospector**](#Prospector) to grow your pipeline and find the right buyer within a named account; all you need is a domain name. Our connector enables your team to automatically send revealed companies back to Clearbit and obtain targeted user profiles, no manual work required.
+
+[**Discover**](#Discover) new accounts that are similar to the profiles in any segment defined in Hull. Leverage our advanced segmentation engine to search through over 20M companies.
+
+The Clearbit connector supports to `create users`, `add traits` and `update traits`.
+
+## Find Your Clearbit API Key
+
+You can find your key in your Clearbit dashboard under the API menu link or by navigating to [https://dashboard.clearbit.com/api](https://dashboard.clearbit.com/api). Clearbit provides two keys, a *secret API key* and a *publishable API key*. Use the **secret API key** to integrate Hull with Clearbit:
+![Find your Clearbit API Key](./docs/clearbitapi01.png)
+
+## Enrichment
+
+The Enrichment feature of the connector allows you to complete a users profile with more than 85 data points provided by Clearbit’s API.
+
+**Configure Enrichment**
+You can enable or disable this feature by toggling the checkbox in the **Enrichment** section of the Settings tab:
+![Enrichment 1](./docs/enrichment01.png)
+
+Enrichment is enabled by default but will only result in API calls to Clearbit’s API if you specify at least one segment in the provided field. You can search segments by simply typing a part of the name into the field or by opening the dropdown via the triangular button.
```

Comment:
@unity and @sbellity, this was not clear to a couple of customers, so we got questions. I am removing it but than we should clarify this generally...